src/App.tsx

```diff
@@ -1,24 +1,264 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+
+import { USER_ID } from './helpers/userId';
+import * as dataOperations from './api/todos';
+
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoError } from './components/TodoError/TodoError';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { Header } from './components/Header/Header';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const [query, setQuery] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isTodoSelected, setIsTodoSelected] = useState('');
+
+  const [isTodoLoading, setIsTodoLoading] = useState<Todo | null>(null);
```

Comment:
Fim similar cases
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig